src/components/folderPageComponents/FolderCard.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Card from '../common/Card';
+import { NonLink } from '../../styles/styledComponents/folderStyled';
+
+export const FolderCard = ({ folder }) => {
+  return (
+    <>
+      <div className="content-wrapper">
+        {folder && folder.length >= 1 ? (
```

Comment:
```suggestion
const hasFolder = folder && folder.length >= 1;
...
        {hasFolder ? (
```

이런식으로 특정 변수에 적절한 네이밍으로 바인딩해주면 변수명으로 로직을 읽을 수 있기 때문에 가독성이 높아집니다.